marlowe-dashboard-client/src/Dashboard/View.purs

```diff
@@ -71,7 +73,13 @@ dashboardScreen currentSlot state =
               [ dashboardBreadcrumb selectedContract
               , main
                   [ classNames [ "relative" ] ] case selectedContractFollowerAppId of
-                  Just followerAppId -> [ renderSubmodule _selectedContract (ContractAction followerAppId) (contractScreen currentSlot) state ]
+                  Just followerAppId ->
+                    [ renderSubmodule
+                        _selectedContract
+                        (ContractAction followerAppId)
+                        (contractScreen { currentSlot, tzOffset, walletDetails, followerAppId })
+                        state
+                    ]
```

Comment:
another tiny thing, but could `let input = { ... }` and condense `renderSubmodule` onto one line